### PR TITLE
sriov: Get PF list by given driver

### PR DESCRIPTION
**Pull request by Yingshun**
Update to get PFs by given driver for scalability testing.

Signed-off-by: Yingshun Cui <yicui@redhat.com>

**Test results:**
_Before the fix:_
` (1/1) type_specific.io-github-autotest-libvirt.sriov.scalability.max_vfs: ERROR: Failed to set vf. (40.83 s)`

_After the fix:_
` (1/1) type_specific.io-github-autotest-libvirt.sriov.scalability.max_vfs: PASS (237.10 s)`

### CODE REVIEW

**Review by dzhengfy — COMMENTED** [review]
Others LGTM

File: libvirt/tests/src/sriov/sriov_scalability.py [hunk 1]
```diff
@@ -43,14 +43,16 @@ def create_network(net_name, pf_name, params):
     libvirt_network.create_or_del_network(net_dict)
 
 
-def get_pf_id_list(pf_info):
+def get_pf_id_list(pf_info, driver):
     """
     Get id of PFs
 
     :param pf_info: Dict, pfs' info
+    :param pf_info: str, pfs' driver
```
[the remark this answers is not in the document]
Comment (Yingshun) [reply libvirt/tests/src/sriov/sriov_scalability.py]: Thanks! Updated.

**Review by dzhengfy — APPROVED** [review]
LGTM

**Review by cliping — APPROVED** [review]
LGTM

**Pull request merged**